verifier/src/lib.rs

```diff
@@ -6,9 +6,12 @@ use groth16_solana::groth16::Groth16Verifyingkey;
 use thiserror::Error;
 
 mod fixture;
+pub use fixture::hash_public_inputs;
 pub use fixture::verify_proof_fixture;
 pub use fixture::SP1ProofFixture;
 
+pub const GROTH16_VK_BYTES: &[u8] = include_bytes!("../vk/groth16_vk.bin");
```

Comment:
IMO we should have the SP1 version in this name since it will change for different versions of SP1, and I'm thinking we may want multiple versions to be supported at once like we do for solidity